sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -122,9 +125,23 @@ def test_label_propagation_closed_form(global_dtype):
     assert_allclose(expected, clf.label_distributions_, atol=1e-4)
 
 
-def test_convergence_speed():
+@pytest.mark.parametrize("accepted_sparse_type", ["sparse_csr", "sparse_csc"])
+@pytest.mark.parametrize("dtype", [np.int32, np.int64])
```

Comment:
Can we change this name to `index_dtype` because it is changing the dtype for the `indices` and the `indptr`?

```suggestion
@pytest.mark.parametrize("index_dtype", [np.int32, np.int64])
```

The rest of the function will need to be updated.